cloudos/jobs/job.py

```diff
@@ -67,7 +71,8 @@ def workflow_id(self, v) -> None:
                 self.cloudos_url,
                 'workflows',
                 self.workspace_id,
-                self.workflow_name)
+                self.workflow_name,
+                self.mainfile)
```

Comment:
@dapineyro I know I need to use `--wdl-importsfile`, because if I don't and during the import the user has defined it, I get an error. 

I am trying to find it in this PR, and I cannot, where is this defined?

<img width="1190" alt="Screenshot 2022-07-20 at 19 41 37" src="https://user-images.githubusercontent.com/38183826/180058498-175f7a75-48b0-41ef-8a84-641d850fa911.png">